index.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>WebDev Summit </title>
+    <link rel="stylesheet" href="./assets/css/style.css">
+</head>
+<body>
+        <!-- Navigation Section -->
+        <header>
+            <nav>
+                <div class="mobile-menu">
+                    <button class="mobile-menu-button">&#9776;</button>
+                  </div>
+                  <div class="mobile-menu-options">
+                    <button class="mobile-menu-closing-btn">
+                        &times;
+                    </button>
+                    <ul class="mobile-menu-list-ul">
+                        <li class="mobile-menu-list-li"><a href="./index.html">Home</a></li>
+                        <li class="mobile-menu-list-li"><a href="./about.html">About</a></li>
+                    </ul>
+                  </div>
+                <div class="logo">
+                    <a href="/Capstone-Project/index.html"><img src="./assets/imgs/logo.png" alt="Logo"></a>
+                </div>
+                <div class="menu">
+                    <ul class="menu-list-ul">
+                        <li class="menu-list-li"><a href="./index.html">Home</a></li>
+                        <li class="menu-list-li"><a href="./about.html">About</a></li>
+                    </ul>
+                </div>
+            </nav>
+        </header>
+        <!-- Main Content Section -->
+        <main>
+        <section class="hero-section">
+            <div class="hero-div">
+                <h1 class="thin-text">"Hello! Sharing Knowledge around the Globe"
+                    <br><h1 class="thick-text">Awareness Summit about web development 2023</h1>
+                </h1>
```

Comment:
- [ ] You should not have more than 1 h1 tag on a page. h1 tags mark the main heading on a page and it should only be one on each page. Please fix this.

Reply:
Resolved